src/views/accounts/AccountChangeSecret/AccountChangeSecretExecution/AccountChangeSecretExecutionList.vue

```diff
@@ -36,10 +36,10 @@ export default {
                 name: 'AccountChangeSecretExecutionDetail',
                 params: { id: row.id }
               }),
+              getTitle: ({ row }) => row['short_id'],
               drawer: true,
               can: this.$hasPerm('accounts.view_changesecretexecution')
-            },
-            width: '240px'
+            }
           },
           change_secret_name: {
             label: this.$t('DisplayName'),
```

Comment:
The provided code does not contain any specific differences, issues or suggestions. However, you may want to ensure that all required parameters are present and there is no typos or formatting mistakes if doing comparisons with previous versions of your application.